core/ui/PageControls/layout.tid

```diff
@@ -0,0 +1,23 @@
+title: $:/core/ui/Buttons/layout
+tags: $:/tags/PageControls
+caption: {{$:/core/images/layout-button}} {{$:/language/Buttons/LayoutSwitcher/Caption}}
+description: {{$:/language/LayoutSwitcher/Description}}
+list-after: $:/core/ui/Buttons/theme
+
+\whitespace trim
+\define control-panel-button(class)
+\whitespace trim
+<$button tooltip={{$:/language/Buttons/LayoutSwitcher/Hint}} aria-label={{$:/language/Buttons/LayoutSwitcher/Caption}} class="""$(tv-config-toolbar-class)$ $class$""">
+{{$:/core/ui/KeyboardShortcuts/switcher}}
+<$list filter="[<tv-config-toolbar-icons>match[yes]]">
+{{$:/core/images/layout-button}}
+</$list>
+<$list filter="[<tv-config-toolbar-text>match[yes]]">
+<span class="tc-btn-text"><$text text={{$:/language/Buttons/LayoutSwitcher/Caption}}/></span>
+</$list>
+</$button>
+\end
+
+<$list filter="[list<tv-story-list>] +[field:title[$:/LayoutSwitcher]]" emptyMessage=<<control-panel-button>>>
```

Comment:
This code looks like it's copied from the page control button for control panel. The intention is to highlight the button if the control panel tiddler is opened in the story river. That doesn't work here because the button action is to trigger the layout switcher. So I think this button can actually look at lot more like a simple one such as $:/core/ui/Buttons/save-wiki

Reply:
Got it, interesting usage, I didn't understand it at first glance, I have to admit...